tests/cli/test_main.py

```
@@ -26,3 +27,27 @@ def test_generate(tmp):
+    # RENDERED output. CI's cli-smoke-test todo block is build/install-only
+    # (D-07), so this is the only gate that catches a future template
+    # import-order (I001) regression in the shipped todo/ package. Running
+    # ``ruff check .`` from the generated dir resolves that project's
+    # ``[tool.ruff]`` tests-excluded scope, exercising the exact shipped gate.
+    argv = ['generate', 'todo-tutorial', tmp.dir, '--defaults']
+
+    with App(argv=argv) as app:
+        app.run()
+
+    result = subprocess.run(
+        ['ruff', 'check', '.'],
```

Ruff (0.15.20)

[error] Starting a process with a partial executable path

(S607)